src/sentry/notifications/helpers.py

```diff
@@ -446,6 +453,39 @@ def get_highest_notification_setting_value(
     return max(notification_settings_by_provider.values(), key=lambda v: v.value)
 
 
+def get_value_for_parent(
```

Comment:
Note to myself as I review, this and `get_value_for_actor` are not related to the bug fix, this is just pulling them out into their own functions.